src/rgw/rgw_rest_swift.cc

```diff
@@ -345,6 +391,12 @@ static void dump_container_metadata(struct req_state *,
 
 void RGWListBucket_ObjStore_SWIFT::send_response()
 {
+  req_state_span ss;
+  char buffer[strlen(__FILENAME__)+strlen(__PRETTY_FUNCTION__)+10];
+  get_span_name(buffer , __FILENAME__,  "function",   __PRETTY_FUNCTION__);
+  start_trace(std::move(ss), {}, s, buffer);
+  set_span_tag(s->root_span, "success", "true");
```

Comment:
i don't think "success" is a useful tag here; we don't know that this is successful, we haven't even started listing

Reply:
I will check this once again

Reply:
success is called in functions like `send_reponse()/_begin()/_send_data()` and these are getting at the last of every request, so if a program reaches there this means it is successful isnt it?